app/views/pages/report.ejs

```diff
@@ -433,7 +433,7 @@
           class="hashtag-item__img h-48 w-48 lg:h-64 lg:w-64 m-2">
       </div>
       <div class="hashtag-item text-center flex flex-col items-center w-1/2">
-        <div class="hashtag-item__title text-cvs-red font-medium text-lg md:text-3xl uppercase m-2">#stay<wbr/>the<wbr/>fuck<wbr/>home</div>
+        <div class="hashtag-item__title text-cvs-red font-medium text-lg md:text-3xl uppercase m-2">#stay<wbr/>indoors</div>
```

Comment:
Why not #stayhome? I think that 's the popular one in The Netherlands? We could make it a translation.

Reply:
Yeah we should probably make it translatable. I think @sbocinec is on it